views/pages/GraphPage.jsx

```diff
@@ -269,10 +213,26 @@ class Graph extends Component {
       openStat: true,
     })
   }
+  fetchGraph = async (subject, study) => {
+    return await window
+      .fetch(`${basePath}/dashboard/${study}/${subject}`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'same-origin',
+      })
+      .then((res) => res.json())
```

Comment:
should we create an API method for this? `api.studySubjects.load(study, subject)`?